### PR TITLE
Scroll updates nav links.

### PR DESCRIPTION
**Pull request by ScottDowne**

### CODE REVIEW

**Comment by ScottDowne** [comment]
Fixes https://github.com/MozillaFoundation/NSF-wireless-innovation-challenge/issues/34

File: source/js/components/sticky-nav.js [hunk 1]
```diff
@@ -24,10 +24,33 @@ var StickyNav = React.createClass({
       active: window.location.hash.replace(`#`, ``) || ``
     };
   },
+  componentDidMount: function() {
+    document.addEventListener(`scroll`, this.onScroll);
+  },
+  componentWillUnmount: function() {
+    document.removeEventListener(`scroll`, this.onScroll);
+  },
+  onScroll: function() {
```
Comment (mmmavis) [inline source/js/components/sticky-nav.js]: If it's possible we should not have things hardcoded here. We should be able to get all nav anchors by ```document.querySelectorAll(`.nav-anchor`)``` and get `id` of each element by ```.getAttribute(`id`)```

Also I think this method doesn't belong to `<StickyNav>` as we are not scrolling within the `<StickyNav>` component itself. We should move this to a page wrapper component (if there's any)

**Review by mmmavis — CHANGES_REQUESTED** [review]

**Comment by mmmavis** [comment]
Um just saw this https://github.com/MozillaFoundation/NSF-wireless-innovation-challenge/issues/17#issuecomment-277762999. What do we do if user starts to scroll when a accordion section is open?

**Comment by ScottDowne** [comment]
I figured it would be an issue but we can land one and fix the other, if it
does in fact conflicts UX wise.

**Comment by mmmavis** [comment]
I personally don't like "update url with anchor as user scrolls", especially when we have so many anchors on the same page. Let me ask in another repo to get some feedback.

File: source/js/components/sticky-nav.js (hunk 1 above)
Comment (ScottDowne) [reply source/js/components/sticky-nav.js]: The wrapper for this is main.js. Not a great place to put something that changes the state of another component. Especially if it's not a react component.

I am going to try moving out the scrolling bits a bit to be a bit more separate from the sticky concept. Hopefully that's enough?

**Review by mmmavis — APPROVED** [review]
looks great to me!

**Pull request merged**